sympy/stats/drv_types.py

```diff
@@ -834,3 +837,85 @@ def Zeta(name, s):
 
     """
     return rv(name, ZetaDistribution, s)
+
+
+#-------------------------------------------------------------------------------
+# Delaporte distribution ------------------------------------------------------------
+
+class DelaporteDistribution(SingleDiscreteDistribution):
+    _argnames = ('lamda', 'alpha', 'beta')
+    set = S.Naturals0
+
+    @staticmethod
+    def check(lamda, alpha, beta):
+        _value_check(lamda > 0, 'lambda should be positive')
+        _value_check(alpha > 0, 'alpha should be positive')
+        _value_check(beta > 0, 'beta should be positive')
+
+    def pdf(self, k):
+        lamda = self.lamda
+        alpha = self.alpha
+        beta = self.beta
+        i = Symbol('i', integer=True)
+        return Sum((gamma(alpha + i)*beta**i*lamda**(k - i)*exp(-lamda)) /
+        (gamma(alpha)*factorial(i)*(1 + beta)**(alpha + i)*factorial(k - i)), (i, 0, k + 1))
+
+    def _moment_generating_function(self, t):
+        return (exp(self.lamda*(exp(t) - 1))) / (1 - self.beta*(exp(t) - 1))**self.alpha
```

Comment:
We should avoid overriding and let `sympy` calculate moment generating function using the generic function defined in the parent class.

Reply:
Moment generating function is overridden for every other distribution in `drv_types` . So should i remove it?